libraries/AP_HAL_ChibiOS/hwdef/F4BY_H743/hwdef.dat

```diff
@@ -0,0 +1,195 @@
+# hw definition file for processing by chibios_hwdef.py
+# for F4BY H743 v3.x.x board description https://f4by.com/en/?doc/fc_f4by_v3.0.1_h743
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID defined in tools board_types.txt as 
+# AP_HW_F4BY_H743
+APJ_BOARD_ID AP_HW_F4BY_H743
+
+# USB setup
+USB_VENDOR 0x27AC # Swift-Flyer
```

Comment:
I'm a bit confused; According to https://usb.org/sites/default/files/vendor_ids051920_0.pdf that ID belongs to "GP Electronics (HK) Limited"